Leanplum-SDK/Classes/Leanplum.h

```diff
@@ -647,41 +647,33 @@ NS_SWIFT_NAME(track(transaction:));
  * any value of your choosing, and will show up in the dashboard.
  * To track a purchase, use LP_PURCHASE_EVENT.
  */
-+ (void)track:(NSString *)event
-NS_SWIFT_NAME(track(event:));
++ (void)track:(NSString *)event;
 
 + (void)track:(NSString *)event
-    withValue:(double)value
-NS_SWIFT_NAME(track(event:value:));
+    withValue:(double)value;
```

Comment:
we should remove `with`.
Adding `NS_SWIFT_NAME(track(_:value:));` will result in `Leanplum.track("my event", value: 5.0)` instead of `Leanplum.track("my event", withValue: 5.0)`